lib/iex/errors/invalid_option_error.rb

```diff
@@ -0,0 +1,12 @@
+module IEX
+  module Errors
+    class InvalidOptionError < StandardError
+      attr_reader :response
+
+      def initialize(response)
+        @response = response
+        super "Invalid option passed into your get chart hash"
```

Comment:
* We should capture the actual option at least in the internals of the error (a field).
* `InvalidOptionError` is a generic name (as it should be) and you can't imply that it was called from get chart. So, the description should simply be `Invalid option` and additional detail would point on the error.